needy/manifest.py

```diff
@@ -0,0 +1,54 @@
+import json
+import logging
+import time
+
+
+class Manifest:
```

Comment:
since this looks like it's only used for caches, if we could organize it so it isn't a top-level module, that'd be really nice